CMakeLists.txt

```diff
@@ -68,8 +69,12 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Config.cmake)
 # add the include path
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+# find doxygen
```

Comment:
You can find Doxygen only if the documentation is enabled, I think that's better